karton/core/main.py

```diff
@@ -10,11 +10,22 @@
 from .__version__ import __version__
 from .backend import KartonBackend
 from .config import Config
-from .karton import Consumer
+from .karton import Consumer, LogConsumer
 
 log = logging.getLogger(__name__)
 
 
+class BasicLogger(LogConsumer):
+    identity = "karton.basic-logger"
```

Comment:
What do you think about a more descriptive identity like `karton.cli-logger`?

Reply:
:+1: renamed